frontend/views/utils/markdown-utils.js

```diff
@@ -127,6 +150,35 @@ export function injectOrStripLink (
   }
 }
 
+export const filterOutInAppLinksFromSafeHTML = (textInSafeHTML: string): Array<Object> => {
```

Comment:
This function finds the temporary string for in-app links, which is generated by `makeInAppLinkElement`, from the text and returns an array of two types of Object; one is for InAppLink, and the other is for original text.